app/javascript/location-hours.js

```diff
@@ -66,14 +66,28 @@ Blacklight.onLoad(function(){
       }
     }
 
+    function getLibraryTimezoneDate(dateString) {
+      const date = new Date(dateString);
+      const formatter = new Intl.DateTimeFormat('en-US', {
+        timeZone: 'America/Los_Angeles',
+        year: 'numeric',
+        month: '2-digit',
+        day: '2-digit',
+        hour: '2-digit',
+        minute: '2-digit',
+        second: '2-digit'
+      });
+      return new Date(formatter.format(date));
```

Comment:
Can we just return the string here?  If you do `new Intl.DateTimeFormat('en', {timeZone: 'America/Los_Angeles', timeStyle: 'short'}).format(date)` then you can simply trim the `:00` and translate the meridian from `AM` to `a`

Reply:
Yes, you are right, no need to jump through extra hoops here.